### ISSUE TITLE
Delete wiki

### ISSUE THREAD

**Issue by rotu** [ros2/ros2#750]
When searching for "Install ROS2" (in Google, Bing, and DuckDuckGo), this ROS2 wiki comes up before the actual content on index.ros.org where it is now hosted. Perhaps this wiki should be deleted altogether to make that official documentation site more discoverable.

Google results:
<img width="773" alt="Untitled" src="https://user-images.githubusercontent.com/119948/62018990-924e9880-b182-11e9-9db1-e75e3612df40.png">

First hit:
<img width="688" alt="Screen Shot 2019-07-28 at 21 57 07" src="https://user-images.githubusercontent.com/119948/62019027-b611de80-b182-11e9-865b-5d51e7dc6262.png">

Second hit:
<img width="868" alt="Screen Shot 2019-07-28 at 21 57 23" src="https://user-images.githubusercontent.com/119948/62019029-b7dba200-b182-11e9-9ef4-bf120beee8e3.png">

**Comment by tfoote** [comment]
I agree it might be time to disable the wiki. We switched all the pages over to be the pointers to the new site in the hopes that people with old links would find their way to the new location and that google would too, since there's no content anymore. However they appear to give more pagerank to the github url than our own even now. 

Since we switched over in October it's now over 9 months later and 2 distributions later it would be better to get the improved search placement over keeping backwards compatibility.
